sw/device/lib/drivers/power_manager/power_manager.h

```diff
@@ -22,11 +22,44 @@ typedef enum power_manager_result {
   kPowerManagerError = 2,
 } power_manager_result_t;
 
+/**
+ * Domains.
+ */
+typedef enum power_manager_sel_domain {
+  kPeriph = 2,
```

Comment:
why 2, 4, 8?
1) they should be generated, 
2) they should have another naming, if you want to use their address then use their address not the enum

Reply:
Solved.

Reply:
Solved.